iron-iconset-svg.html

```diff
@@ -156,12 +166,21 @@
      */
     _targetIsRTL: function(target) {
       if (this.__targetIsRTL == null) {
-        if (target && target.nodeType !== Node.ELEMENT_NODE) {
-          target = target.host;
-        }
+        if (this.useGlobalRtlAttribute) {
+          var globalElement =
+              (document.body && document.body.hasAttribute('dir'))
+                  ? document.body
+                  : document.documentElement;
```

Comment:
is documentElement ever null?

Reply:
is it `html`? the og comment promised we also look at `html`

Reply:
`document.documentElement` is the `<HTML>` node and should never be null in any reasonable circumstance AFAIK.